src/main/java/com/bravo/user/validator/UserValidator.java

```diff
@@ -15,6 +15,16 @@ public void validateId(String id){
       throw new BadRequestException("'id' is required");
     }
   }
+  public void validateName(String name){
+    if(ValidatorUtil.isInvalid(name)){
+      throw new BadRequestException("'name' is required");
+    }
+    if(!name.matches("[A-Za-z]*")){
+      throw new BadRequestException("'name' cannot contain numbers, spaces, or special characters");
```

Comment:
Like that you validated name separate from the other methods.  Should the greedy match be a ? here and not a * to avoid an empty string?

Reply:
In practice, an empty string would not make it here, since the previous line would throw a "Name is Required" error.  A `?` will match exactly one, I've switched this to a `+` and added beginning and end of line characters so it expects the name to contain 1 or more alpha characters.